lib/paper_trail/config.rb

```diff
@@ -32,7 +32,9 @@ def serialized_attributes=(value)
     end
 
     def track_associations
-      @track_associations ||= PaperTrail::VersionAssociation.table_exists?
+      @track_associations.nil? ?
+        PaperTrail::VersionAssociation.table_exists? :
+        @track_associations
```

Comment:
We could actually simplify this to `@track_associations && PaperTrail::VersionAssociation.table_exists?`, right?  Not as explicitly obvious what is going on, but my thought is that this will actually prevent errors in scenarios where users try to turn the feature on without having the underlying required table.  Alternatively we could raise an error or a warning in those scenarios. Thoughts?